django/test/testcases.py

```diff
@@ -549,8 +558,18 @@ def assertNotContains(
             response, text, status_code, msg_prefix, html
         )
 
+        content = (
+            b"".join(response.streaming_content)
+            if response.streaming
+            else response.content
+        )
+
         self.assertEqual(
-            real_count, 0, msg_prefix + "Response should not contain %s" % text_repr
+            real_count,
+            0,
+            msg_prefix
+            + "%s unexpectedly found in the following response\n%s"
```

Comment:
Nice that this is now more consistent with Python’s `assertIn` and Django’s `assertTemplateNotUsed`! I wish we used simpler language than "unexpectedly" but consistency matters more.